framework/src/userobject/GeneralUserObject.C

```diff
@@ -20,6 +20,8 @@ validParams<GeneralUserObject>()
 {
   InputParameters params = validParams<UserObject>();
   params += validParams<MaterialPropertyInterface>();
+  params.addParam<bool>(
+      "force_preaux", false, "Forces the GeneralUserObject to be executed in PREAUX");
```

Comment:
Lets add an `addParamNamesToGroup` call here and put this new parameter in the "Advanced" group.